distributed/worker.py

```diff
@@ -3472,6 +3489,7 @@ async def execute(self, key: str, *, stimulus_id: str) -> None:
             assert ts, self.story(key)
             ts.done = True
             result["key"] = ts.key
+            result["stimulus_id"] = f"{result['op']}-{time()}"
```

Comment:
I went with with the following approach 

```python
result["stimulus_id"] = stimulus_id = f"{result['op']}-{time()}"
```

I think it's not  always possible to  have access to `new_stimulus_id` in the exception block since the generation of the `new_stimulus_id` actually occurs inside the try block.

Reply:
Different stimuli are now generated on success `f"{result['op']}-{time()}"` and failure conditions `f"execute-erred-{time()}`.